airbyte-integrations/connectors/source-bing-ads/source_bing_ads/base_streams.py

```diff
@@ -206,13 +206,21 @@ def request_params(
             "ReturnAdditionalFields": self.additional_fields,
         }
 
+    def _transform_tax_fields(self, record: Mapping[str, Any]) -> Mapping[str, Any]:
+        tax_certificates = record["TaxCertificate"].get("TaxCertificates", {}) if record.get("TaxCertificate") is not None else {}
+        if tax_certificates and not isinstance(tax_certificates, list):
```

Comment:
Should this transformation be applied for each item in the list if `isinstance(tax_certificates, list) is True`?

Reply:
Thanks for explanation!